docs/integrations/amazon-aws/sns.md

```diff
@@ -141,6 +146,24 @@ Scope (Specific Data): account=* eventname eventsource \"sns.amazonaws.com\"
 ```
 
 
+## Centralized AWS CloudTrail Log Collection
+In case you have a centralized collection of CloudTrail logs and are ingesting them from all accounts into a single Sumo Logic CloudTrail log source, create the following **Field Extraction Rule** to map a proper AWS account(s) friendly name/alias. Create it if not already present/update it as required.
+
+* **Rule Name**: AWS Accounts
```

Comment:
We should describe these fields.

Reply:
@nmohantyctr : It's important to describe these fields.

Reply:
created asana ticket for you, @nmohantyctr https://app.asana.com/0/1183741177809740/1204058079587177